apps/music/index.html

```diff
@@ -110,7 +111,7 @@ <h1 id="title-text" data-l10n-id="music"></h1>
               <button class="rating-star" data-rating="4" role="radio" data-l10n-id="rating-star" data-l10n-args='{"n":"4"}'></button>
               <button class="rating-star" data-rating="5" role="radio" data-l10n-id="rating-star" data-l10n-args='{"n":"5"}'></button>
             </div>
-            <button id="player-album-shuffle" class="player-cover-button" data-icon="shuffle"></button>
+            <button id="player-album-shuffle" class="player-cover-button" data-icon="shuffle" aria-pressed="true"></button>
```

Comment:
I'm not sure what `aria-pressed` does here, but I'll assume you know what you're doing...